wrappers/python/nntile/nntile_core.cc

```diff
@@ -205,13 +205,14 @@ void def_mod_tile(py::module_ &m)
         def("index_to_linear", &TileTraits::index_to_linear);
     // Define wrappers for Tile<T>
     def_class_tile<fp32_t>(m, "Tile_fp32");
+    def_class_tile<fp32_fast_tf32_t>(m, "Tile_fp32_fast_tf32");
     def_class_tile<fp64_t>(m, "Tile_fp64");
 }
 
 // numpy.ndarray -> Tensor
 template<typename T>
 void tensor_from_array(const tensor::Tensor<T> &tensor,
-        const py::array_t<T, py::array::f_style | py::array::forcecast> &array)
+        const py::array_t<T> &array)
```

Comment:
why did you remove `forcecast` and `f_style` flags? Without these flags copying data from numpy will be erroneous if user provides C-style input.